bench-tps/src/main.rs

```diff
@@ -103,10 +103,12 @@ fn create_connection_cache(
     let (stake, total_stake) =
         find_node_activated_stake(rpc_client, client_node_id.pubkey()).unwrap_or_default();
     info!("Stake for specified client_node_id: {stake}, total stake: {total_stake}");
-    let stakes = HashMap::from([
+    let stakes = [
         (client_node_id.pubkey(), stake),
         (Pubkey::new_unique(), total_stake - stake),
-    ]);
+    ]
+    .into_iter()
+    .collect::<HashMap<_, _, solana_pubkey::PubkeyHasherBuilder>>();
```

Comment:
nit: Please import PubkeyHasherBuilder.

(and in the other files too, please)

Reply:
yes. 5ec01f5318b0a467dc5bbd85b86db86afd9bc0c7